damus/Views/ConfigView.swift

```diff
@@ -21,6 +21,10 @@ struct ConfigView: View {
     @State var relays: [RelayDescriptor]
     @EnvironmentObject var user_settings: UserSettingsStore
     
+    /// True = poll will display votes from everyone, false = poll will display votes from friends only (and friends of friends)
+    /// This option is included because of the potential problem with bots.
+    @AppStorage("poll_results_everyone") var poll_results_everyone: Bool = false
```

Comment:
I've never used AppStorage before. Does this just save it in UserSettings? I eventually want to scope user settings by pubkey so maybe its best to have it in there